profiles/base.yaml

```diff
@@ -817,6 +817,7 @@ consumable_lockboxes:
 
 # pick.lic settings for live boxes
 # https://elanthipedia.play.net/Lich_script_repository#pick
+never_blind_pick: false
 stop_pick_on_mindlock: true
 pick_blind: false
```

Comment:
Could we leverage the existing `pick_blind: false` setting instead of add a new setting?

Reply:
No, this should be "always_blind_pick".